Source/WebCore/platform/graphics/displaylists/DisplayListItems.h

```diff
@@ -626,6 +653,8 @@ class DrawPattern {
     const AffineTransform& patternTransform() const { return m_patternTransform; }
     FloatPoint phase() const { return m_phase; }
     FloatSize spacing() const { return m_spacing; }
+    const ImagePaintingOptions& options() const { return m_options; }
```

Comment:
`ImagePaintingOptions` seems to be designed to pass by value, so we probably don't need to return a const reference here.